sql/src/main/java/io/crate/analyze/WindowDefinition.java

```diff
@@ -54,12 +65,17 @@ public WindowDefinition(StreamInput in) throws IOException {
 
     public WindowDefinition(List<Symbol> partitions,
                             @Nullable OrderBy orderBy,
-                            @Nullable WindowFrameDefinition windowFrameDefinition) {
+                            @Nonnull WindowFrameDefinition windowFrameDefinition) {
         this.partitions = partitions;
         this.orderBy = orderBy;
         this.windowFrameDefinition = windowFrameDefinition;
     }
 
+    public WindowDefinition(List<Symbol> partitions,
```

Comment:
Would it be possible to keep a single constructor where the windowFrameDefinition is nullable and set to the default if null?